app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,17 @@
+module SessionsHelper
```

Comment:
Change this file from Spaces: 4 to Spaces: 2.
Ask me questions if you don't know how or this is confusing :bow: